### PR TITLE
new chrome API to add a quickstart to a namespace

### PR DESCRIPTION
**Pull request by radekkaluzik**
https://issues.redhat.com/browse/RHCLOUD-20677

### CODE REVIEW

File: src/js/redux/index.ts [hunk 1]
```diff
@@ -66,6 +68,7 @@ import {
 } from './action-types';
 import { ChromeState, GlobalFilterState, ReduxState } from './store';
 import { AnyAction } from 'redux';
+//import { addQuickstart } from './actions';
```
Comment (Hyperkid123) [inline src/js/redux/index.ts:71]: extra comment

File: src/js/redux/reducers.ts [hunk 2]
```diff
@@ -250,6 +250,19 @@ export function populateQuickstartsReducer(
   };
 }
 
+export function addQuickstartstoApp(state: ChromeState, { app, quickstart }: { app: string; quickstart: QuickStart }) {
```
Comment (Hyperkid123) [inline src/js/redux/reducers.ts:253]: Can you write some tests for the function?

File: src/js/redux/index.ts (hunk 1 above)
Comment (radekkaluzik) [reply src/js/redux/index.ts]: removed

File: src/js/redux/reducers.ts (hunk 2 above)
Comment (radekkaluzik) [reply src/js/redux/reducers.ts:249]: done

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**